api/live.js

```diff
@@ -51,15 +55,38 @@ async function handleContrib(userId, req, res) {
         personId: userId,
         taskId: input.taskId
       }).save();
-      return res.json({ contribution: result });
+      return res.json({
+        contribution: result,
+        message: `Successfully created a task contribution for ${profile.firstName} ${profile.lastName}`
+      });
     } catch (e) {
-      return res.status(500).json({ error: e.message });
+      return res.status(500).json({ message: e.message });
     }
   }
 }
 
 async function handleCheckin(userId, req, res) {
   try {
+    const profile = await models.HackerProfile.findOne({
```

Comment:
moved this chunk of code up here because we want to do a status check before we attempt to create anything else